official/resnet/resnet_run_loop.py

```diff
@@ -287,34 +309,37 @@ def exclude_batch_norm(name):
       eval_metric_ops=metrics)
 
 
-def validate_batch_size_for_multi_gpu(batch_size):
+def compute_per_device_batch_size(batch_size, use_distribution_strategy,
+                                  gpus_for_distribution_strategy):
   """For multi-gpu, batch-size must be a multiple of the number of GPUs.
 
-  Note that this should eventually be handled by replicate_model_fn
+  Note that this should eventually be handled by DistributionStrategies
   directly. Multi-GPU support is currently experimental, however,
   so doing the work here until that feature is in place.
 
   Args:
-    batch_size: the number of examples processed in each training batch.
+    batch_size: Global batch size to be divided among devices.
```

Comment:
Maybe add something like, This should be equal to num_gpus times the single-GPU batch_size.